### DISCUSSION TITLE
.hex file

### DISCUSSION THREAD

**Discussion by parascodes007** [erichelgeson/BlueSCSI-v1#38] [Q&A] [answered]
Where to find the '.hex' file Blue SCSI.

**Comment by erichelgeson** [comment, marked as answer]
Under the GitHub releases you can download the bin file. Hope that gets you what you need.

**Reply by parascodes007** [reply]
After we load the software, the LED blinks 5 times and after some interval when I reinsert the micro SD card same process goes on and the computer does not show the drive. Please suggest how to rectify this problem.

**Reply by erichelgeson** [reply]
Please take a look at the troubleshooting guide in the wiki on 5x blinking - that indicates the bluepill does not see the SD Card. The troubleshooting guide is quite thorough and please follow all steps.
